string goal
---
uint8 navigation_status
